cloudevents/bindings/amqp-protocol-binding.md

```diff
@@ -55,9 +55,10 @@ format][message-format].
 
 ### 1.3. Content Modes
 
-The specification defines two content modes for transferring events:
-_structured_ and _binary_. Every compliant implementation SHOULD support both
-modes.
+The specification defines three content modes for transferring events:
+_structured_, _binary_ and _batch_. The AMQP protocol binding does not currently
+support the batch content mode. Every compliant implementation SHOULD support
```

Comment:
it's too bad we didn't think more about this before since the SHOULD here doesn't guarantee interop if two impls don't support the same mode. I would have loved to make this a MUST for binary and structured. Oh well.